borsaco/telegram-bot-api-bundle/1.0/config/packages/telegram.yaml

```diff
@@ -0,0 +1,29 @@
+telegram_bot_api:
```

Comment:
It looks like you added all possible configuration here. 

You should add the minumum required config and then link to your bundle's readme.